src/OrchardCore.Modules/OrchardCore.Redis/Startup.cs

```diff
@@ -56,6 +66,10 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IModularTenantEvents>(sp => sp.GetRequiredService<IRedisService>());
             services.AddSingleton<IRedisDatabaseFactory, RedisDatabaseFactory>();
         }
+
+        // expect the certificate to exist, but not much else
+        private static bool AlwaysValidateCertificate(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors) =>
```

Comment:
@ovekaaven 

I don't understand that comment. Can you please change it to state what it supposed to do?

Also, start the comment with capital letter and end it with a period.

Place `=> ....;` on a new line too.

Reply:
Done.